spec/vmpooler/api/v1_spec.rb

```diff
@@ -180,6 +180,7 @@ def app()
 
   describe '/vm' do
     let(:redis)  { double('redis') }
+    let(:statsd)  { double('stats') }
```

Comment:
If we have a class representing statsd we could actually just subclass is in testing and neuter the publication / record in the statsd object that things were to be published.  It's the same as a double or mock, in many ways, but just mentioning it (because "such mock")